react-graph/src/PointNode.tsx

```diff
@@ -126,25 +128,31 @@ export class PointNode {
      * @param {number} point - The point to retrieve from the tree
      */
     public GetPoint(point: number): [...number[]] {
+        return this.GetPoints(point, 1)[0];
+    }
+
+    /**
+     * Retrieves a specified number of points from the PointNode tree, centered around a point
+     * @param {number} point - The center point of the point retrieval.
+     * @param {number} pointsRetrieved - The number of points to retrieve
+     */
+    public GetPoints(point: number, pointsRetrieved = 1, bucketLowerNeighbor?: PointNode, bucketUpperNeighbor?: PointNode): [...number[]][] {
+        if (bucketUpperNeighbor === undefined && bucketLowerNeighbor === undefined && pointsRetrieved > MaxPoints)
+            console.warn("Points requested exceeds maximum per bucket. This may result in an unexpected output.");
```

Comment:
what is this warning supposed to prevent? Why can't we request more points than are available in a single bucket?

Reply:
Well, technically you can request up to double before issues may start to appear. Main one being lets say you pick a point near the edge of a bucket, it will spill over to the neighbor, but not to the neighbor after that if there is more than double a buckets worth of points requested. The spill over would stop after the first neighbor bucket, despite not enough points being returned. It would still work, but not return the expected number.

Reply:
shouldn't we fix that rather than add a warning?